profilegen/src/profile_template_files.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+#
+# Module that stores various template files to be used
+# in the generation of profile SDKs
+#
+module ProfileTemplateFiles
+  SPEC_TEMPLATE   = './res/templates/spec_template.template'
```

Comment:
let's name it as `resources` instead of `res`

Reply:
Done